i18n/zh/docusaurus-plugin-content-docs/version-v2.8.0/developers/protocol.md

```
@@ -3,35 +3,39 @@ title: 协议设计
 translated: true
 ---
 
-## 设备注册
+### 设备注册
```

markdownlint-cli2 (0.23.2)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)